src/st/class-wpml-pb-loader.php

```diff
@@ -12,6 +12,22 @@ public function __construct(
 
 		$page_builder_strategies = array();
 
+		/**
+		 * This filter hook provide the API page builders names that need to be supported.
+		 *
+		 * For each PB name, we will create a dedicated strategy and a proper string package namespace.
+		 *
+		 * It's called in 2 places:
+		 * - `WPML_Page_Builders_Integration` for external plugins
+		 * - `WPML_Gutenberg_Integration` for WordPress Core block editor
+		 *
+		 * @param string[] Required plugin names (e.g. `Beaver Builder`, `Gutenberg`)
```

Comment:
@kkarpieszuk this parameter is never passed: why you are documenting it as "required"?

I would change it into `string[] Not used`.

Then, in the description, you can add something like "it returns the plugin names (e.g. `Beaver Builder`, `Gutenberg`)".

Reply:
@andreasciamanna, in my understanding the first parameter is the value we actually want to filter (an empty `array` by default).

I used "Required" as an adjective of "plugin" (i.e. "The name of the plugins that are required".

I am a bit confused. Maybe it would be faster if you paste here the PHPDoc that you have in mind.